openedx/core/djangoapps/theming/helpers.py

```diff
@@ -17,7 +17,16 @@ def get_value(val_name, default=None, **kwargs):
     """
     This is a proxy function to hide microsite_configuration behind comprehensive theming.
     """
-    return microsite.get_value(val_name, default=default, **kwargs)
+    micro_value = microsite.get_value(val_name, default=default, **kwargs)
+    if not default:
+        default_value = settings.val_name
```

Comment:
we may need to protect against AttributeNotFound here, if no parameter is defined in the settings object -- maybe consider using `getattr()`